05/snake_for_if.py

```diff
@@ -0,0 +1,16 @@
+pets_start_second = []
+pets_sorted = []
+pets = ['had', 'andulka', 'pes', 'kočka', 'králík']
+copy_pets = pets
+
+for pet in pets:
+    pet = pet[0] + pet[1:]
```

Comment:
Aký je účel tohoto riadku? Rozpisuješ napr. slovo "had" na "h" a "ad" , to dáš dohromady a pripraďuješ to do premennej ktorá je vlastne to isté :)